src/Raven.Server/ServerWide/Commands/CompareExchangeCommands.cs

```diff
@@ -112,29 +165,70 @@ public static object ConvertResult(JsonOperationContext ctx, object result)
 
     public class RemoveCompareExchangeCommand : CompareExchangeCommandBase
     {
+        private bool _fromBackup;
+
         public RemoveCompareExchangeCommand() { }
-        public RemoveCompareExchangeCommand(string key, string database, long index, JsonOperationContext contextToReturnResult) : base(key, database, index, contextToReturnResult) { }
+        public RemoveCompareExchangeCommand(string database, string key, long index, JsonOperationContext contextToReturnResult) : base(database, key, index, contextToReturnResult) { }
```

Comment:
double-check all the places where we are using this ctor

Reply:
done